pom.xml

```diff
@@ -17,22 +17,7 @@
   <description>Open source platform for continuous inspection of code quality</description>
 
   <modules>
-    <module>sonar-plugin-api-deps</module>
-    <module>sonar-application</module>
-    <module>sonar-scanner-engine</module>
-    <module>sonar-scanner-protocol</module>
-    <module>sonar-scanner-engine-shaded</module>
-    <module>sonar-check-api</module>
-    <module>sonar-core</module>
-    <module>sonar-duplications</module>
-    <module>sonar-home</module>
-    <module>sonar-markdown</module>
-    <module>sonar-plugin-api</module>
     <module>server</module>
```

Comment:
This file should be dropped, especially as the module "server" does not exist anymore.